tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{36,37,38}-test{,-alldeps,-devdeps}{,-cov}
-    py{36,37,38}-test-numpy{116,117,118}
-    py{36,37,38}-test-astropy{30,40,lts}
+    py{37,38}-test{,-alldeps,-devdeps}{,-cov}
+    py{37,38}-test-numpy{117,118}
+    py{37,38}-test-astropy{lts}
```

Comment:
I rebased but there is no conflict yet because #563 is not merged. We can merge this first and I can rebase the other one too. Up to you, @rosteen and @javerbukh .